databases/week3/resturant with errors.sql

```diff
@@ -0,0 +1,151 @@
+USE persianMeal;
+
+-- Get all meals
+ SELECT * FROM meal;
+ 
+-- Add new meal
```

Comment:
when you use id ad autoincrement, than we dont need to defined hear.

Reply:
thats reason you are having error while inserting values and show duplicating value in primary key.